stdlib/websub/src/main/ballerina/websub/Module.md

```diff
@@ -255,3 +255,99 @@ where the values specified via the Config API would override values specified as
 | b7a.websub.hub.remotepublish   | false         | Whether publishing updates against the topics in the hub could be done by remote publishers via HTTP requests with `hub.mode` set to `publish`  |
 | b7a.websub.hub.topicregistration | true      | Whether a topic needs to be registered at the hub for publishers to publish updates against the topic and for subscribers to send subscription requests for the topic |
 | b7a.websub.hub.enablessl | true      | Whether the Hub service should be exposed over HTTPS |
+
+## Introducing Specific Subscriber Services
+
+Ballerina's WebSub subscriber service endpoint can be extended to introduce specific Webhooks.
+ 
+With specific subscriber services, multiple resources corresponding to the possible notifications for a particular 
```

Comment:
Passive voice doesn't work so well in this case as it makes it harder to understand. Something like this with the subject first.

"With specific subscriber services, you can introduce multiple resources corresponding to the possible notifications for a particular topic, instead of the single `onNotification` resource."

Alternatively, you could start the sentence with "You". 

In general use active voice more where the subject (in this case "You" comes first).

Reply:
IINM, it seems we haven't used active voice in the Module.md files. I've updated the content here, but still not active voice since it might seem inconsistent, WDYT?

Reply:
Yes, you're right. Module.md files are mostly in passive voice. We used that approach because we wanted to be more direct and less conversational there. I think you can use passive in this case as well. My bad.